.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,18 @@
+---
+name: User story
+about: Create a user story and add it to the tracker
```

Comment:
What is the purpose of the `tracker`?

Reply:
So far we didn't have a label identifying the user story issues so we were using the `tracker` label. Those are expected to have a set of sub-tasks. 

I think we should explore the option of having a label `kind/userstory`

Reply:
@devguyio What would be the difference between `kind/userstory` and `tracker`? Because otherwise I think we might as well stick to `tracker` or rename it to `kind/userstory` so it updates the old ones.

Reply:
I believe that tracker was introduced in ODH to track generic "tracker" issues that are not necessarily user stories such as opendatahub-io/opendatahub-community#107 . 

A `kind/userstory` IS a user story.  If we created that label, we'd definitely relabel older user stories with the new label instead of `tracker`.

Reply:
idk @LaVLaS @adelton what do you like more `tracker` or `kind/userstory`?

Reply:
`userstory` describes what the content of the specific issue is about. `tracker` sounds like some internal "process" thing. I can get some sense what `userstory` means. For `tracker`, not so much.

If I'm an external contributor, I would ignore the `tracker` label, it does not convey any useful information to people outside of the project.

Reply:
@LaVLaS how do you feel about renaming `tracker` to `kind/userstory`? It should update for all of the past user stories too.